doc/multisig-tutorial.md

```diff
@@ -109,7 +109,7 @@ Then import the descriptors created in the previous step using the `importdescri
 After that, `getwalletinfo` can be used to check if the wallet was created successfully.
 
 ```bash
-./src/bitcoin-cli -signet -named createwallet wallet_name="multisig_wallet_01" disable_private_keys=true blank=true descriptors=true
+./src/bitcoin-cli -signet -named createwallet "multisig_wallet_01" true true
```

Comment:
I like the named syntax better here. `true true` doesn't say much.

Reply:
Thanks, I hesitated on this and prefer the named version as well. Done.